addons/website/migrations/11.0.1.0/post-migration.py

```diff
@@ -68,6 +68,28 @@ def add_website_homepages(env):
     )
 
 
+def update_menu_urls(env):
+    """Update some menu URLs that must change."""
+    # Fix menus that include the "website." prefix
+    menus = env["website.menu"].search([
+        ("url", "=like", "/page/website.%"),
+    ])
+    for menu in menus:
+        menu.url = menu.url.replace("website.", "", 1)
+    # Fix menus pointing to "/page/homepage"
+    menus = env["website.menu"].search([
+        "|", "|", "|", "|", "|",
+        ("url", "=", "/page/homepage"),
+        ("url", "=", "/page/homepage/"),
+        ("url", "=like", "/page/homepage?%"),
+        ("url", "=like", "/page/homepage/?%"),
+        ("url", "=like", "/page/homepage/#%"),
```

Comment:
For this case you will have as result `//#%`

Reply:
Fixed also, see above.